### PR TITLE
[Carbondata-390] float data type supported

### PR DESCRIPTION
**Pull request by rahulforallp**
Float data type supported with spark 1.6 and spark 2.0.

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/374/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/375/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.5.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/379/

File: core/src/main/java/org/apache/carbondata/core/datastorage/store/compression/nondecimal/UnCompressNonDecimalByte.java [hunk 1]
```diff
@@ -85,12 +85,15 @@
   }
 
   @Override public CarbonReadDataHolder getValues(int decimal, Object maxValueObject) {
```
Comment (ravipesala) [inline core/src/main/java/org/apache/carbondata/core/datastorage/store/compression/nondecimal/UnCompressNonDecimalByte.java]: This is not the way you supposed to do, please check how long data type is supported.

**Comment by ravipesala** [comment]
rename title from [Carbondata 390] to [CARBONDATA-390]

**Comment by ravipesala** [comment]
@rahulforallp  Thank you for working on it. 
But another PR https://github.com/apache/incubator-carbondata/pull/521 with simple changes are merged to master, so please close this PR

**Comment by rahulforallp** [comment]
Ok, thanks for you support

**Pull request closed without merging**